templates/documentation/delivery-analytics/private-video-get-started.md

```diff
@@ -1,9 +1,10 @@
 ---
 title: "Get started with private videos"
+slug: private-video-get-started
```

Comment:
`slug` can be removed, as confirmed by Doctave [here](https://api-video.slack.com/archives/C05DTNR3QDS/p1697618392282579)

Reply:
Done ✅